setup.py

```diff
@@ -16,5 +16,5 @@
     packages=setuptools.find_packages(),
     install_requires=[
         "ipfsapi==0.4.3", "py-evm==0.2.0a34", "py-solc==3.2.0",
-        "schematics==2.1.0", "web3==4.8.2", "yapf==0.25.0"
+        "schematics==2.1.0", "web3==4.8.2", "yapf==0.25.0", "mypy==0.670"
```

Comment:
You can probably pull schematics as that now should be in `hmt-basemodels`

Reply:
Good point, fixed!